src/orange/OrangeParams.hh

```diff
@@ -20,6 +20,8 @@
 #include "BoundingBox.hh"
 #include "OrangeData.hh"
 #include "OrangeTypes.hh"
+#include "detail/RectArrayInserter.hh"
+#include "detail/UnitIndexer.hh"
```

Comment:
Don't include implementation details in the header file. The header should only have classes necessary for the interface. Move these to the `.cc` file.